Machine readable documents requirements (#424)
w3c/wot-binding-templates: registry-requirements.md

    - Form Vocabulary Definition as Table
    - Default and possible mappings to operations as a Table
  - Examples

> - <a name="req-docs">Req-Docs</a>: The requirement for the machine-readable documents are as follows:

Reviewer: ```suggestion
- <a name="req-docs">Req-Docs</a>: The requirements for the machine-readable documents are as follows:
```

  - There MUST be a JSON Schema (version to be defined) that allows validating the elements added by the entry.

>   - There SHOULD be a JSON-LD Context and an ontology for the entry.

Author: ```suggestion
  - There SHOULD be a JSON-LD Context and an ontology for the entry. Note that the lack of JSON-LD Context creates an RDF representation that will probably cause issues in RDF databases.

```

>   - There MAY be other documents such as code, diagrams, standalone examples, which are helpful for the implementers.

Reviewer: ```suggestion
  - There MAY be other documents which are helpful to implementers, such as code, diagrams, and/or standalone examples.
```

  - These documents MUST be available to the reviewer.
  - The reviewer MUST include these documents in their review.

>   - All documents associated to the registry entry MUST have the same version string, i.e. all documents should update their version when there is a change.

Reviewer: ```suggestion
  - All documents associated with the registry entry MUST have the same version string, i.e., the versions of all associated documents MUST be updated when there is a change to any of them.
```
